### ISSUE TITLE
Using polygon tool, with "drawOnSingleClick: true", drawing inside existing annotation can't be started with a single click

### ISSUE THREAD

**Issue by IvanVas** [recogito/annotorious-openseadragon#164]
Hi,

Using polygon tool, with "drawOnSingleClick: true", drawing inside existing annotation can't be started with a single click.
This is in OSD plugin.
In Annotorious this works correctly.

Also, not sure if this is connected:
The rectangle tool does not work with click, drag only, both on free space & inside the annotation.

**Comment by rsimon** [comment]
Yes, that's currently an unresolved issue. Single click will always select in OSD at the moment. It's on the list, but I can't say for sure when I'll get a chance to look into it.
